### DISCUSSION TITLE
顶部图标消失，只有长条图，左侧插件工具栏也没有

### DISCUSSION THREAD

**Discussion by SyberRabbit** [the1812/Bilibili-Evolved#5022] [提问 / Q&A] [answered]
Edge：130.0.2849.80、win11-24H2、
除了油猴，其他插件及脚本全部关闭：使用jsDelivr 、GitHub 、开关梯子，各种组合都试过了
都是如图所示：
![Uploading 20241109221930126.png…]()
图片貌似上传不了，我就先用图床链接了
![](https://dl.img.timecdn.cn/2024/11/09/20241109221930126.png!h.webp)

**Comment by Geeyun-JY3** [comment, marked as answer]
别问，问就是重复：#4476

**Reply by SyberRabbit** [reply]
感谢指路

**Comment by Young1324** [comment]
我也是没有左侧工具栏，但我有上面的导航。我的解决方法是在chrome上卸载油猴，然后在edge上下载暴力猴（另一个插件管理工具），重新安装插件，然后问题就解决了。估计是新版油猴和插件不适配

**Reply by fvrings** [reply]
非常好的评论，使我的插件旋转
